ordered_associative_task/student/src/impl.cpp

```diff
@@ -1 +1,6 @@
 #include "impl.h"
+std::map<size_t,size_t> mapConverter={{29,1},{49,2},{69,3},{89,4},{SIZE_MAX,5}};
+
+size_t convertMarkTo5PointSystem(size_t markIn100Point){
+     return mapConverter.lower_bound(markIn100Point)->second;
```

Comment:
what if lower_bound will return end iterator?

Reply:
Since you cannot pass a size_t value greater than size_max, I thought that whatever value you pass to lower_bound never returns an end iterator

Reply:
hacky way, I'd rely on such mapping only:
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};